packages/web/app/src/lib/fetch-json.ts

```
@@ -0,0 +1,134 @@
+export namespace FetchJsonErrors {
+  export type FetchJsonErrors = FetchJsonResponseErrors | FetchJsonRequestErrors;
+
+  // --------------------------------
+  // Response Error Classes
+  // --------------------------------
+
+  export type FetchJsonRequestErrors = FetchJsonRequestTypeError | FetchJsonRequestNetworkError;
+
+  export class FetchJsonRequestNetworkError extends Kit.Errors.ContextualError<
+    'FetchJsonRequestNetworkError',
+    {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)